python/raft-ann-bench/src/raft-ann-bench/run/conf/nytimes-256-inner.json

```diff
@@ -0,0 +1,1352 @@
+{
+  "dataset": {
+    "name": "nytimes-256-inner",
+    "base_file": "nytimes-256-inner/base.fbin",
+    "query_file": "nytimes-256-inner/query.fbin",
+    "groundtruth_neighbors_file": "nytimes-256-inner/groundtruth.neighbors.ibin",
+    "distance": "euclidean"
+  },
+  "search_basic_param": {
+    "batch_size": 5000,
+    "k": 10,
+    "run_count": 3
+  },
+  "index": [
+    {
+      "name" : "hnswlib.M12",
+      "algo" : "hnswlib",
+      "build_param": {"M":12, "efConstruction":500, "numThreads":32},
+      "file" : "index/nytimes-256-inner/hnswlib/M12",
+      "search_params" : [
+        {"ef":10, "numThreads":1},
+        {"ef":20, "numThreads":1},
+        {"ef":40, "numThreads":1},
+        {"ef":60, "numThreads":1},
+        {"ef":80, "numThreads":1},
+        {"ef":120, "numThreads":1},
+        {"ef":200, "numThreads":1},
+        {"ef":400, "numThreads":1},
+        {"ef":600, "numThreads":1},
+        {"ef":800, "numThreads":1}
+      ],
+      "search_result_file" : "result/nytimes-256-inner/hnswlib/M12"
```

Comment:
This key is not needed anymore, it was necessary in the old benchmarks

Reply:
we still have it in a lot of files, perhaps I can do a follow up to remove it everywhere?

Reply:
Please!